### ISSUE TITLE
I've been getting the afk timer in the last couple days since the update to steam gameplay

### ISSUE THREAD

**Issue by MysophobiaSquared** [Eternita-S/AntiAfkKick#24]
Hi, just like the title says, ever since SE messed with steam logins, I've been getting the anti-afk timer.

**Comment by Eternita-S** [comment]
are you using dalamud plugin or standalone/act version?

**Comment by MysophobiaSquared** [comment]
> are you using dalamud plugin or standalone/act version?

I've been using the standalone version. I didnt know there was a dalamud plugin version

**Comment by Eternita-S** [comment]
are you possibly running ffxiv or steam or xivlauncher as admin?

**Comment by MysophobiaSquared** [comment]
I'll check again in a bit. Will update if Im having the same issue as Im not entirely sure

**Comment by Eternita-S** [comment]
well if you're running game as admin, program needs to be ran as admin as well. You don't really need to run game as admin, though.

**Comment by MysophobiaSquared** [comment]
Yeah, after running both as admin and running one or the other as admin, it
still doesnt work when using steam play

**Comment by Eternita-S** [comment]
Are you maybe using any controllers?

**Comment by MysophobiaSquared** [comment]
No controllers

**Comment by MysophobiaSquared** [comment]
![image](https://user-images.githubusercontent.com/102254311/160218495-a0856da3-d27d-4ea9-94b4-27a345ecf444.png)


Seemingly my system is noticing idling is fine.

**Comment by Eternita-S** [comment]
okay, I will assume "system is not idle" appeared when you touched mouse or keyboard.
Can you right click tray icon, click "invoke manually" and send screenshot of the result then?

**Comment by MysophobiaSquared** [comment]
![image](https://user-images.githubusercontent.com/102254311/160585514-68319fae-5eeb-4755-ba22-258f7d77370a.png)


Sorry about the delay, I was out of town. 

This is what I get

**Comment by MysophobiaSquared** [comment]
That seems to have fixed it. I dunno why, seeing as I've never had to do that before.

**Comment by MysophobiaSquared** [comment]
What is the name of the dalamud plugin version?
